tests/test_noise_models.py

```diff
@@ -38,6 +40,8 @@ def add_chrom_noise_to_model(model):
     model["TNCHROMAMP"].quantity = 1e-14
```

Comment:
Isn't this too small?

Reply:
Yes, you are right. So, I check git blame...   and it seems you committed that line @abhisrkckl , haha :p

Reply:
Given that 0 is much larger than -14, this value is actually super large

Reply:
😭